fractional-NFT/.gitignore

```diff
@@ -0,0 +1,3 @@
+target
+project/fractional-NFT-contract/out
+project/token-distributor-contract/out

```

Comment:
I would remove all other `.gitignore`'s and add the `out` directories in this root `.gitignore`. The `target`s in other `.gitignore`s can be omitted since cargo workspace creates a single `target` directory for every member of the workspace.